Formula/mtm.rb

```diff
@@ -0,0 +1,21 @@
+class Mtm < Formula
+  desc "Perhaps the smallest useful terminal multiplexer in the world"
+  homepage "https://github.com/deadpixi/mtm"
+  url "https://github.com/deadpixi/mtm/archive/1.0.1.tar.gz"
+  sha256 "cb1758d810860d25c7dc6d6d5440ad79055a22935f521be7d7d9fae40124add8"
+
+  depends_on "ncurses"
+
+  def install
+    system "make"
+    bin.install "mtm"
+    man1.install "mtm.1"
+  end
+
+  test do
+    # mtm doesn't provide any immediately returning switches,
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.